cmd/dmverity-vhd/main.go

```diff
@@ -28,13 +28,14 @@ const usage = `dmverity-vhd is a command line tool for creating LCOW layer VHDs
 const (
 	usernameFlag       = "username"
 	passwordFlag       = "password"
-	imageFlag          = "image"
+	inputFlag          = "input"
```

Comment:
instead of repurposing `imageFlag`, wouldn't this work better with `tarballFlag` instead?

Reply:
The imageFlag is required for the CreateVHD command, so even if we don't want to create a VHD from a container image, (we want to use the tarball), we'd still have to provide that flag as it is currently structured. Unless you want to make the image flag optional too (it is also used in the roothashcommand and is required).

Reply:
ah, ok. this makes sense. maybe later we can look into making a mutually exclusive required flags or something. but definitely not in scope for this PR.